packages/core/src/browser/browser-clipboard-service.ts

```diff
@@ -19,6 +19,7 @@ import { isFirefox } from './browser';
 import { ClipboardService } from './clipboard-service';
 import { ILogger } from '../common/logger';
 import { MessageService } from '../common/message-service';
+import { nls } from 'src/common/nls';
```

Comment:
This will fail the linter:

```suggestion
import { nls } from '../common/nls';
```